sql/4_2_2-to-4_3_1_upgrade.sql

```diff
@@ -708,6 +708,8 @@ ALTER TABLE drugs
 #IfNotRow2D list_options list_id lists option_id page_validation
 INSERT INTO `list_options` (`list_id`, `option_id`, `title`, `seq`) VALUES ('lists', 'page_validation', 'Page Validation', 298);
 INSERT INTO `list_options` (`list_id`, `option_id`, `title`, `seq`, `notes`, `activity`) VALUES ('page_validation', 'add_edit_issue#theform', '/interface/patient_file/summary/add_edit_issue.php', 10, '{form_title:{presence: true}}', 0);
+INSERT INTO `list_options` (`list_id`, `option_id`, `title`, `seq`, `notes`, `activity`) VALUES ('page_validation', 'encounters#new_encounter', '/interface/forms/newpatient/new.php', 20, '{pc_catid:{exclusion: ["_blank"]}}', 0);
```

Comment:
No. Because there are many forms that only have a name but no id, we chose to use the name of the form in the method implementation and not the id. Otherwise we would have to go around and add id to all forms. Is this ok?